test/extended/compatibility.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Runs extended compatibility tests with a previous version
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+source "${OS_ROOT}/test/extended/setup.sh"
+
+# Previous version to test against
+PREVIOUS_VERSION="v1.2.0"
+
+if [[ -n "${PREVIOUS_API:-}" ]]; then
```

Comment:
this doesn't give me what I want ... previous_api is just a flag ("1" or "[empty]"). Based on that value, I want API_SERVER_VERSION to either be blank or the same value as PREVIOUS_VERSION.